docs/cow-protocol/tutorials/arbitrate/orderbook.md

```diff
@@ -0,0 +1,120 @@
+---
+sidebar_position: 1
+---
+
+# Orderbook
+
+The orderbook is the main API that Traders, UIs and other integrations use to interact with CoW Protocol.
+
+Its implementation can be found [here](https://github.com/cowprotocol/services/tree/main/crates/orderbook). The API is documented in detail [here](../../reference/apis/orderbook). 
+
+## Overview
+
+By and large the orderbook is a [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) service for the state of the protocol.
+It connects to the database which is shared with the (autopilot)[./autopilot] to insert or update orders and serves information about existing orders as well as previous trades and auctions.
+
+## Architecture
+
+The standard Trader interaction flow can be seen in the following sequence diagram:
+
+sequenceDiagram
+    actor User
+    participant Orderbook
+    participant Solver1
+    activate Orderbook
+    User->>Orderbook: POST /quote
+    Orderbook->>SolverN: quote?
+    activate SolverN
+    Orderbook->>Solver1: quote?
+    activate Solver1
+    Solver1->>Solver1: compute matching for<br> "single order batch"
+    SolverN->>SolverN: compute matching for<br> "single order batch"
+    Solver1-->>Orderbook: result
+    deactivate Solver1
+    SolverN-->>Orderbook: result
+    deactivate SolverN
+    Orderbook->>Orderbook: simulate quote
+    Orderbook->>Orderbook: pick best result
+    Orderbook->>Database: store quote
+    Orderbook-->>User: quote
+    deactivate Orderbook
+    User->>User: 🖊 Sign order
+    User->>Orderbook: POST /order
+    activate Orderbook
+    Orderbook->>Orderbook: verify order
+    Orderbook->>Database: fetch quote
+    opt if quote not found
+      Orderbook->>Solver1: quote?
+      Solver1-->>Orderbook: result
+      Orderbook->>SolverN: quote?
+      SolverN-->>Orderbook: result
+      Orderbook->>Orderbook: smualate & pick
+    end
+    Orderbook->>Database: insert order
+    Orderbook-->>User: order UID
+    deactivate Orderbook
+    User->>User: ⏳ Wait for happy moo
+    User->>Orderbook: GET /trades
+    activate Orderbook
+    Orderbook->>Database: lookup order
+    Database-->>Orderbook: order & trades
+    Orderbook-->>User: trades
+    deactivate Orderbook
+
+After selecting the parameters of their trade, most Traders want to see an estimate of how much tokens they will receive in order to pick a reasonable limit price.
+For that they send a `/quote` request to the orderbook. 
+The orderbook is connected to a subset of solvers that are participating in the competitions (technically their (**drivers**)[./driver]) and creates price estimates by sending these solvers a request to solve a batch auction containing just a single order.
```

Comment:
Not sure we should mention the implementation detail that we are connected to the `drivers`.